### DISCUSSION TITLE
Displays won't stay off

### DISCUSSION THREAD

**Discussion by sandorex** [xanderfrangos/twinkle-tray#278] [Q&A] [answered]
Hello i have problem with my monitors waking up automatically after few minutes every time i turn them off

**Comment by xanderfrangos** [comment, marked as answer]
Hey @sandorex,

By default, the "Turn Off Displays" action only turns off the monitors via software, as it's the most compatible. The software signal will display a black screen until input (ex. mouse or keyboard) is detected by Windows. The hardware signal will turn off power to the monitors. 

Try adjusting the settings for this to "Hardware* & software signal" and see if that turns off your monitors completely.

![Screenshot 2021-11-21 210323](https://user-images.githubusercontent.com/33106561/142790451-9f3759c5-0628-4771-b0d1-a2bc11da512c.png)

**Reply by sandorex** [reply]
That's cool but i want the standby feature but for some reason it doesn't stay off even without me touching the pc

**Reply by xanderfrangos** [reply]
Then it sounds like your mouse or another input device is waking the PC up. My wireless mouse will frequently wake up my PC even though it's perfectly still, unless I turn it off first.
